ILRepack/ILRepack.cs

```diff
@@ -240,16 +241,25 @@ public static AssemblyName GetRepackAssemblyName(IEnumerable<AssemblyName> repac
             return repackAssemblyNames?.FirstOrDefault(name => name.Name == repackedAssemblyName) ?? fallbackType.Assembly.GetName();
         }
 
-        void PrintRepackVersion()
+        void PrintRepackHeader()
         {
             var assemblies = GetRepackAssemblyNames(typeof(ILRepack));
             var ilRepack = GetRepackAssemblyName(assemblies, "ILRepack", typeof(ILRepack));
             Logger.Info($"IL Repack - Version {ilRepack.Version.ToString(3)}");
             Logger.Verbose($"Runtime: {typeof(ILRepack).Assembly.FullName}");
-            foreach (var asb in assemblies)
-            {
-                Logger.Verbose($" - {asb.FullName}");
-            }
+            Logger.Info("---------------------------------------------------");
+
+            var assembliesArgument =
+            Options.InputAssemblies.Aggregate(
+                string.Empty,
+                (previous, item) => previous + ' ' + item);
+
+            Logger.Info("Arguments: " +
+                        $"/out:{Options.OutputFile} " +
```

Comment:
this is a bit partial, while fine for v1, would fit better in a toString/toCommandLine method in option type, no ?